### DISCUSSION TITLE
Installation Error | Need Help

### DISCUSSION THREAD

**Discussion by irtiza7** [skiptools/skip#279] [Q&A] [answered]
I have installed the latest Skiptools and my XCode is also the latest i.e. 16.1
I am using Intel Macbook Pro 2019 with latest Android Studio (Android Studio Ladybug | 2024.2.1 Patch 1)

Also I have started this installation process 2 days ago. So I dont think my initial trial period is expired.

When i run `skip checkup`
I get the following errors:

skip checkup
[✓] Skip version 1.1.18 (= 1.1.18)
[✓] macOS version 15.1 (> 13.5.0)
[!] Error running sysctl (Intel)
[✓] Swift version 6.0.2 (> 5.9.0)
[✓] Xcode version 16.1 (> 15.0.0)
[✓] Xcode tools SDKs: 7
[✓] Homebrew version 4.4.3 (> 4.1.0)
[✓] Gradle version 8.10.2 (> 8.6.0)
[✓] Java version 23 (> 21.0.0)
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 2024.2
[✓] Android tools SDKs: 6
[✓] Resolve dependencies (17.14s)
[✓] Build hello-skip (26.93s)
[✓] Test Swift (17.63s)
[✓] Test Kotlin (46.1s)
[✗] Archive iOS ipa (9.46s)
[✗] The file “Payload” doesn’t exist.
[✗] Skip 1.1.18 checkup failed with 2 errors
Output logged to: /tmp/skip-checkup-2024-10-31T18:59:30Z.txt
See https://skip.tools/docs/faq and https://community.skip.tools for help and 1 warning
Error: 3 errors

**Comment by marcprux** [comment, marked as answer]
Can you look in the output log (`/tmp/skip-checkup-2024-10-31T18:59:30Z.txt`) and see if you spot any error messages?

**Reply by irtiza7** [reply]
Yeah the logs helped. Basically the problem was that the XCode had to download some files (e.g. iPhone simulators) after it got updated with system update.
